<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerEditDialog</class>
 <widget class="QDialog" name="ComputerEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="add_Photo_computer_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_edit_Year_Built">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_edit_if_original">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_Year_Built">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>51</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Built :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_edit_if_developed">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Developed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_add_computer_info">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add info:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable_Computer_Development">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Development :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_Type">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_computer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_edit_Input_Type">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_add_computer_photo">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add photo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_edit_Name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_comptuer_name">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>23</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_Add_Info">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_photo">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
